shared/src/commonMain/kotlin/co/nimblehq/kmm/showcase/data/JobDetails.kt

```diff
@@ -0,0 +1,100 @@
+package co.nimblehq.kmm.showcase.data
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class Data(
+    @SerialName("data") val data: JobDetails
+)
+
+@Serializable
+data class JobDetails(
```

Comment:
Should be in singular, `JobDetail`

Reply:
@luongvo So `JobDetailsFragment` should be `JobDetailFragment`?